specification/2.0/README.md

```diff
@@ -179,11 +179,13 @@ To simplify client-side implementation, glTF has following restrictions on JSON
 
 ## URIs
 
-glTF uses URIs to reference buffers and image resources. These URIs may point to external resources or be data URIs that embed resources in the JSON. Embedded resources use "data" URI scheme ([RFC2397](https://tools.ietf.org/html/rfc2397)).
- 
- > **Implementation Note:** Data URIs could be [decoded with JavaScript](https://developer.mozilla.org/en-US/docs/Web/API/WindowBase64/Base64_encoding_and_decoding) or consumed directly by web browsers in HTML tags.
+glTF uses URIs to reference buffers and image resources. Clients must support at least these two URI types:
+
+- **Data URIs** that embed resources in the JSON. They use syntax defined by [RFC&nbsp;2397](https://tools.ietf.org/html/rfc2397).
+  > **Implementation Note:** Data URIs could be [decoded with JavaScript](https://developer.mozilla.org/en-US/docs/Web/API/WindowBase64/Base64_encoding_and_decoding) or consumed directly by web browsers in HTML tags.
 
-Client implementations are required to support only embedded resources and relative external references (in a sense of [RFC3986](https://tools.ietf.org/html/rfc3986#section-4.2)). Clients are free to support other schemes (such as `http://`) depending on expected usage.
+- **Relative references** as defined by [RFC&nbsp;3986, Section 4.2](https://tools.ietf.org/html/rfc3986#section-4.2) with additional restriction that only `path-noscheme` part is allowed (i.e., path must not start with `/`, query and fragment parts aren't allowed). All reserved characters must be percent-encoded.
```

Comment:
I'm not quite clear on how to interpret this — we're saying that these things (absolute URI schemes, query and fragment parameters, ...) are invalid glTF, but (according to implementation note below) that clients can choose to support them anyway?

Or do you mean to say, clients must support _at least_ relative URLs that don't include query/fragment, but may also choose to support more?

Reply:
This one:
> Or do you mean to say, clients must support _at least_ relative URLs that don't include query/fragment, but may also choose to support more? And therefore authoring a glTF with query/fragment is not invalid, just less portable?

However, I think that usage of query/fragment should issue validation notice. Could you refine language so that intention is clear?

Reply:
How about:

* **Relative URI paths** — or `path-noscheme` as defined by RFC 3986, [Section 4.2](https://tools.ietf.org/html/rfc3986#section-4.2) — without scheme, authority, or parameters. Reserved characters must be percent-encoded, per RFC 3986, [Section 2.2](https://tools.ietf.org/html/rfc3986#section-2.2).

  > **Implementation Note:** Clients can optionally support additional URI components. For example `http://` or `file://` schemes, authorities/hostnames, absolute paths, and query or fragment parameters. Assets containing these additional URI components may be less portable.